### ISSUE TITLE
2TAP feedback fragment is clipped in landscape

### ISSUE THREAD

**Issue by LukasPaczos** [mapbox/mapbox-navigation-android#3029]
The feedback fragment is clipped at the bottom in the landscape mode on the Pixel 2 XL.
![device-2020-05-25-202522](https://user-images.githubusercontent.com/16925074/82836127-4a91be00-9ec6-11ea-9d18-66ed60eac486.png)
![device-2020-05-25-202543](https://user-images.githubusercontent.com/16925074/82836130-4c5b8180-9ec6-11ea-88a7-780ce7a6fff1.png)

/cc @JunDai

**Comment by Guardiola31337** [comment]
Also the font color is messed up (white instead of blue). Great catch!

**Comment by JunDai** [comment]
thanks for reporting this. It's not only happening on Pixel 2XL but all the devices.

we discussed this before I started working on this. and the decision was to not consider the landscape mode at that time.
I will tag @Guardiola31337 and @LukasPaczos in the original ticket.

But I think we should support it given that seems we are having demo to customer which cares landscape mode, right?

It's better to have UE team involved into this. @zugaldia / @asinghal22 , wdyt?
